doc/source/cd/methods/cvmdrift.ipynb

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
<p>"emphirical" -&gt; "empirical"</p><p>Whilst the expression for omega^2 is of course correct I think integrals and measures etc are unnecessary in this case and some users would more easily understand the discrete sum version.</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>